code/game/objects/items/cards_ids.dm

```diff
@@ -729,7 +729,7 @@
 			. += "The [linked_dept.account_holder] linked to the ID reports a balance of [linked_dept.account_balance] cr."
 
 	if(HAS_TRAIT(user, TRAIT_ID_APPRAISER))
-		. += HAS_TRAIT(src, TRAIT_JOB_FIRST_ID_CARD) ? span_boldnotice("Hmm... yes, this ID was issued from Central Command!") : span_boldnotice("This ID was created in this sector, not by Central Command.")
+		. += HAS_TRAIT(src, TRAIT_JOB_FIRST_ID_CARD) ? span_boldnotice("Hmm... yes, this ID was issued from [command_name()]!") : span_boldnotice("This ID was created in this sector, not by [command_name()].")
```

Comment:
why are we updating this in this PR? this should be atomized out into its own verbose PR because it just doesn't fit in with the rest of the changes in this PR and should be discussed in its own task-specific PR.

Reply:
Mainly because I came across it, but you're right, it doesn't really fit. I'll remove it.